site/layouts/partials/head.html

```diff
@@ -49,49 +49,11 @@
 <link rel="stylesheet" href="{{ "css/note.css" | absURL }}">
 
 <script async defer src="https://buttons.github.io/buttons.js"></script>
-<script async src="//cdn.bizible.com/scripts/bizible.js"></script>
-<!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-64458817-2"></script>
-<script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag() {dataLayer.push(arguments);}
-    gtag('js', new Date());
-    gtag('config', 'UA-64458817-2');
-</script>
 
-<!-- Marketo Munchkin Tracking code for Marketing Team -->
```

Comment:
To double check: Are you sure these aren't used any longer?

Reply:
Yes, they are still in use. I have moved them to the GTM container. If you inspect network connections when you load the page you can see the various calls are still there.